src/bin/cargo.rs

```diff
@@ -51,6 +51,7 @@ Options:
 
 Some common cargo commands are (see all commands with --list):
     build       Compile the current project
+    check       Check the current project without compiling
```

Comment:
This description is weaker because it re-uses 'check'. And it _does_ compile, in a sense....

I'll try to come up with a suggestion for better words later.